srcpkgs/sunshine/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'sunshine'
+pkgname=sunshine
+version=0.23.1
+revision=1
+build_style=cmake
+configure_args="-Wno-dev -DBUILD_TESTS=OFF -DSUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine
+ -DSUNSHINE_ASSETS_DIR=/usr/share/sunshine"
+hostmakedepends="boost cmake git make nodejs pkg-config"
+makedepends="boost-devel doxygen graphviz libayatana-appindicator-devel libboost_filesystem
+ libboost_locale libboost_program_options libcap-devel libcurl-devel libdrm-devel libevdev-devel
+ libnuma-devel libnotify-devel libopusenc-devel libva-devel libvdpau-devel libX11-devel
+ libxcb-devel libXcursor-devel libXfixes-devel libXi-devel libXinerama-devel libXrandr-devel
+ libXtst-devel MesaLib-devel miniupnpc-devel nv-codec-headers opus-devel openssl-devel
+ pulseaudio-devel wayland-protocols"
+short_desc="Self-hosted GameStream host for Moonlight"
+maintainer="myothk <myoothkyaw@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://app.lizardbyte.dev"
+_repo="https://github.com/LizardByte/Sunshine.git"
+do_fetch() {
+	[ -d "${pkgname}-${version}" ] && rm -rf "${pkgname}-${version}"
+	git clone ${_repo} --branch=v${version} --single-branch --recurse-submodules "${pkgname}-${version}"
```

Comment:
don't use git to fetch

if you need to get submodules, add them as distfiles and move them into place with `vsrcextract`